README.md

```diff
@@ -121,7 +121,10 @@ You may use the --enter or --descend options to list the contents of, or recursi
     tag --list --enter .
     tag --list --descend .
     tag -d .
+    
+To get a list of all tags in the current directory you can combine `tag` with `tr` (to split multiple tags from one file to a seperate line), `sort`, and `uniq` (which removes duplicate entries):
 
+    tag --no-name * | tr "," "\n" | sort | uniq
 	
```

Comment:
And the unique flag to sort also shortens your command:

`tag -Ng * | sort -u`

While this example shows how you can combine the usage of tag with various other shell tools, I'm not sure that it's appropriate for the Readme.